ts/go/cmd/gatekeeper_validate/gatekeeper_validate.go

```diff
@@ -15,8 +15,8 @@
 package main
```

Comment:
This functions is no longer needed.

Reply:
yeah, I added a comment about cleaning up the ts/go as your previous PR https://github.com/GoogleContainerTools/kpt-functions-sdk/pull/331 Or would you want to merge that PR? I can rebase mine.

Reply:
Saw your comments. I'll clean up the code in this PR